### PR TITLE
add formatting to htmx source

### PR DESCRIPTION
**Pull request by t3dotgg**
## Description
I saw that the HTMX source didn't include a formatter, thought it might be useful to add one!

## Checklist
...

### CODE REVIEW

**Comment by ssebastianoo** [comment]
LGTM

**Comment by DJAruun** [comment]
Finally some good looking code!

**Comment by GamerGirlandCo** [comment]
looks good to me!! :3

**Comment by LeoMiguelB** [comment]
nice one, I approve

**Comment by VampireChicken12** [comment]
LGTM

**Comment by steveruu** [comment]
LGTM 🚀

**Comment by SimulShift** [comment]
Nit: this PR is too well done. Maybe make it a little  well done.

**Comment by markorusic** [comment]
amazing effort!

**Comment by Benzolio** [comment]
Consistent formatting is always a good thing, right? ...Right?

**Comment by maxrzaw** [comment]
LGTM!

**Comment by gurgelio** [comment]
all tests are passing!!!

**Comment by gsporto** [comment]
LGTM

**Comment by Yash-Singh1** [comment]
<section align=right>

**Review by itsnotv — COMMENTED** [review]
LGTM

**Review by risingsunomi — COMMENTED** [review]
LGTM

**Comment by Visual917** [comment]
LGTM

**Comment by nilswloewen** [comment]
Denied. 

Please integrate this into the build step so we can be sure the minified version is properly formatted as well. Standards matter, and we have a reputation to uphold. 

(I'm the CEO of HTMX by the way)

**Comment by lightleviathan0** [comment]
lgtm!

**Comment by fawzii0x3** [comment]
Lgtm

**Comment by lukemorales** [comment]
seal of approval
LGTM!

**Comment by TheBearodactyl** [comment]
Revolutionary 
![xQc claps fast [lWDpYps-kZM].png](https://github.com/bigskysoftware/htmx/assets/114454115/4b27621d-c8ab-4ca0-8282-dd87281536b5)

**Comment by TheBearodactyl** [comment]
Revolutionary
https://github.com/bigskysoftware/htmx/assets/114454115/fb5274bd-9361-4597-8b70-42a63105b6a3

**Comment by alexpetros** [comment]
> I saw that the HTMX source didn't include a formatter, thought it might be useful to add one!

This is incorrect: https://github.com/bigskysoftware/htmx/blob/f4f448db0a413e403d3c7f627d9e6d2b356bf0d8/package.json#L31

**Comment by Sainan** [comment]
Here I was thinking the way `prettier` formats code is awful. But I guess in an ecosystem as vast as NPM, there's always ought to be a worse package.

**Comment by none23** [comment]
It's a shame, this the PR has been closed.
The readability improvements are so obvious

**Review by SahilMahale — COMMENTED** [review]
LGTM

File: package.json [hunk 1]
```diff
@@ -74,6 +74,7 @@
     "mocha-chrome": "https://github.com/Telroshan/mocha-chrome",
     "mocha-webdriver": "^0.3.2",
     "mock-socket": "^9.3.1",
+    "shittier": "0.1.1",
```
Comment (none23) [inline package.json:77]:
```suggestion
    "shittier": "^0.1.1",
```

**Comment by chrisbward** [comment]
didn't even look, but as CEO I sign this off, lgtm

**Comment by andrewdoro** [comment]
still better than adding TS

**Comment by cl1ents** [comment]
needs more loud snake case

**Review by proudmuslim-dev — COMMENTED** [review]
LGTM!

**Pull request closed without merging**